util/tracing/detect/detect.go

```diff
@@ -82,6 +83,10 @@ func detect() error {
 	if exp == nil {
 		return nil
 	}
+
+	// enable log with traceID when valid exporter
+	bklog.EnableLogWithTraceID()
```

Comment:
Little bit weird still that this isn't called in `main`. But I guess it is ok as `detect` itself should only be called in `main`.

Reply:
in `main`, we still couldn't know the `Exporter` is ready or not.

so i have to enable log with trace id when setup exporter.